plugins/websocket/src/InitMe.cpp

```diff
@@ -23,26 +23,26 @@
 #include <neurala/plugin/PluginStatus.h>
 #include <neurala/utils/Version.h>
 
-#include "Discoverer.h"
-#include "Input.h"
-#include "Output.h"
+#include "websocket/Discoverer.h"
+#include "websocket/Input.h"
+#include "websocket/Output.h"
 
 extern "C" PLUGIN_API NeuralaPluginExitFunction
 initMe(NeuralaPluginManager* pluginManager, std::error_code* status)
 {
 	using namespace neurala;
 	auto& pm = *dynamic_cast<PluginRegistrar*>(pluginManager);
-	*status = pm.registerPlugin<plug::ws::Discoverer>("websocketDiscoverer", Version(1, 0));
+	*status = pm.registerPlugin<plug::ws::Discoverer>("libwebsocketDiscoverer", Version(1, 0));
 	if (*status != PluginStatus::success())
 	{
 		return nullptr;
 	}
-	*status = pm.registerPlugin<plug::ws::Input>("Input", Version(1, 0));
+	*status = pm.registerPlugin<plug::ws::Input>("websocketInput", Version(1, 0));
 	if (*status != PluginStatus::success())
 	{
 		return nullptr;
 	}
-	*status = pm.registerPlugin<plug::ws::Output>("Output", Version(1, 0));
+	*status = pm.registerPlugin<plug::ws::Output>("websocketOutput", Version(1, 0));
```

Comment:
Why is the first one `libwebsocket`, and the other are only `websocket`?

Reply:
Looking at `src\video\VideoPluginDiscoverer.cpp` now, I don't see the reason, though this is how I got it working at the time the changes were made.

I couldn't get the Inspector to work properly for testing yesterday, I am trying again today. If your environment is set up for it, could you please verify the plugin as well?

Reply:
Fixed the naming (now it works fine) and changed the socket number (for whatever reason the old one crashes the acceptor on my PC). After that I was able to test it; it is ready to merge in my opinion.